python_ta/reporters/templates/stylesheet.css

```diff
@@ -197,26 +292,26 @@ section {
 }
 
 .code-error-id {
-  color: rgba(219, 0, 57, 0.9);
+  color: var(--code-error);
 }
 
 .code-error-id a:link {
 }
 
 .style-error-id {
-  color: rgb(0, 111, 255);
+  color: var(--style-error);
 }
 
 .style-error-id a:link {
 }
 
 .line-num {
-  color: #2d3441;
+  color: var(--text-secondary);
   font-size: 90%;
 }
 
 .grey {
-  color: rgb(87, 87, 87);
+  color: var(--text-muted);
 }
 
 .gbold {
```

Comment:
I liked you adding a variable for this style, you should just make it a different one from `--text-muted` (and make sure it looks good for both light and dark modes)